### DISCUSSION TITLE
Edit page redirect

### DISCUSSION THREAD

**Discussion by dimdam** [filamentphp/filament#1272] [Help] [answered]
### Package

filament/filament

### Package Version

v2.9.3

### Laravel Version

8.6

### Livewire Version

2.9

### PHP Version

8.1.1

### Bug description

When creating a new record , i get redirected to the edit page . How can stop that or redirect to the list page for example?

### Steps to reproduce

Create a record for example a new user (fill all the necessary info such as password,name,etc).

Press the create button.

Edit page appears.

**Comment by danharrin** [comment, marked as answer]
You may specify a custom redirect URL for the Create and Edit pages by overriding the `getRedirectUrl()` method.

For example, the Create form can redirect back to the List page when it is submitted:

```php
protected function getRedirectUrl(): ?string
{
    return $this->getResource()::getUrl('index');
}
```

**Reply by dimdam** [reply]
I'm getting this

` Call to undefined method App\Filament\Resources\UserResource::generateUrl() `

**Reply by danharrin** [reply]
Try `getUrl()`

**Reply by dimdam** [reply]
Yes,this seems to work.Thank you.
